.ado/templates/prepare-env.yml

```diff
@@ -23,16 +23,16 @@ steps:
         Get-ChildItem "${env:ProgramFiles(x86)}\Microsoft SDKs\UWPNuGetPackages\runtime.win10-x86.Microsoft.Net.Native.SharedLibrary" | Select-Object FullName
         Get-ChildItem "${env:ProgramFiles(x86)}\Microsoft SDKs\UWPNuGetPackages\Microsoft.Net.UWPCoreRuntimeSdk" | Select-Object FullName
 
-  # The commit tag in the nuspec requires that we use at least nuget 5.8 (because things break with nuget versions before and Vs 16.8 or later)
-  - task: NuGetToolInstaller@0
-    inputs:
-      versionSpec: ">=5.8.0"
-
   - task: NodeTool@0
     displayName: Installing Node
     inputs:
       versionSpec: '14.x' 
 
+  # The commit tag in the nuspec requires that we use at least nuget 5.8 (because things break with nuget versions before and Vs 16.8 or later)
+  - task: NuGetToolInstaller@0
+    inputs:
+      versionSpec: ">=5.8.0"
+
```

Comment:
NuGetToolInstaller relies on Node. WinDev pool I think is using old version of node that doesn't recognize the opts, so we need to set Node version earlier.

Reply:
Ergg... It seems to use Node 10 regardless for this on them